src/pages/EditOrderPage.js

```diff
@@ -169,6 +174,16 @@ const EditOrderPage = () => {
     setDeliveryDate(parseDateToEdit(order.deliveryDate));
     setDeliveryMethod(order.deliveryMethod);
 
+    if (order.deliveryFee > 0) {
+      setDeliveryPrice(order.deliveryFee);
+    } else {
+      setDeliveryPrice(orderDeliveryFee);
+    }
+
+    if (order.haveExtraDeliveryFee) {
+      setHaveExtraFee(true);
+    }
```

Comment:
we can use this setter with the Boolean of `order.haveExtraDeliveryFee` since it will be set to true if truthy or false if falsy

```suggestion
      setHaveExtraFee(!!order.haveExtraDeliveryFee);
```

Reply:
woooooowww :exploding_head: